libcoap/src/error.rs

```diff
@@ -82,16 +83,25 @@ pub enum OptionValueError {
     /// A string value could not be converted to UTF-8.
     #[error("CoAP option has invalid value: invalid string")]
     StringConversion(#[from] FromUtf8Error),
+    /// A string value could not be converted to UTF-8.
```

Comment:
Documentation seems to be incorrect.
```suggestion
    /// A URI could not be parsed.
```